app/assets/stylesheets/cart.scss

```
@@ -14,7 +14,7 @@
     gap: 40px;
     padding: 15px 0;
 
-    img {
+    .cart-image-size {
```

ページ毎にsassでネストしてstyleを適用しているので個人的には特定のclass名を付与する必要はないようにも思います🤔
class名を付与した意図は何でしょうか？（他のscssファイルについても）

---

```suggestion
    img {
```
class名ではなくimgタグにstyleを効かす元の実装でいいと思います。